website/docs/docs/core/create-a-python-virtual-environment.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Create a Python Virtual Environment
```

Comment:
our [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#titles) mentions we should use sentence case, not title case
```suggestion
title: Create a python virtual environment
```